styles.css

```diff
@@ -75,3 +75,274 @@ ul {
   list-style-type: none;
   padding-left: 0;
 }
+
+/*
+-------------------------------------------
+Works section for the Portfolio begins here
+-------------------------------------------
+*/
+
+.works-wrapper {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  margin-left: 24px;
+  margin-right: 24px;
+}
+
+.card-container {
+  margin-top: 114px;
+  padding: 16px;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  background: #fff;
+  height: 552px;
+}
+
+/* .card2-container {
+  margin-top: 88px;
+  padding: 16px;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  background: #fff;
+}
+
+.card3-container {
+  margin-top: 88px;
+  margin-bottom: 88px;
+  padding: 16px;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  background: #fff;
+}
+
+.card4-container {
+  padding: 16px;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  background: #fff;
+} */
```

Comment:
[OPTIONAL]
- [ ] Kindly delete this piece of commented code, as it seems inappropriate for the overall look and organization of your code.